service/test/_registry_test.js

```diff
@@ -87,10 +90,13 @@ function testDevicesNonBooleanForce(methodUnderTest, argName, argValue, Expected
 
 function testWrongLengthArg(methodUnderTest, argName, argValue, ExpectedErrorType) {
   var errorName = ExpectedErrorType ? ExpectedErrorType.name : 'Error';
-  it('throws a ' + errorName + ' if \'' + argName + '\' is length \'' + argValue.length + '\' (type:' + typeof(argValue) + ')', function() {
-    var registry = new Registry({ host: 'host', sharedAccessSignature: 'sas' });
-    assert.throws(function() {
-      registry[methodUnderTest](argValue, true, function() {});
```

Comment:
>true,  [](start = 42, length = 6)

Why gone?

Reply:
None of the method tested by this helper method (e.g. add/update/removeDevice) takes a boolean as a parameter.

Reply:
Won't fix.

---
In reply to: [222829194](https://github.com/Azure/azure-iot-sdk-node/pull/372#discussion_r222829194) [](ancestors = 222829194)